full-service-bin/mainnet/start-full-service-offline.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright (c) 2018-2021 MobileCoin Inc.
+#
+# Launches a local `full-service` instance that syncs the ledger from two nodes in the
+# network and hosts a wallet service running on port 9090
+
+set -e
+
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+pushd "$(dirname "$0")"
+
+killall full-service || true
+
+LEDGER_DB_DIR="$1"
+WALLET_DB_DIR="$2"
+WALLET_DB_FILE="$3"
+
+mkdir -p "${LEDGER_DB_DIR}"
+mkdir -p "${WALLET_DB_DIR}"
+
+echo "Starting full-service with ${LEDGER_DB_DIR} and ${WALLET_DB_DIR} and ${WALLET_DB_FILE}" > /tmp/mylog
+
+RUST_LOG=debug,mc_connection=error,mc_ledger_sync=error ./full-service \
```

Comment:
The same name is used for the testnet and mainnet full service files? That could lead to problems, couldn't it?

Reply:
nope, it's fine :)

Reply:
also this was supposed to be a PR into main, not develop for some README changes I made directly in the repo, so it was showing 3000+ lines of changes that were not accurate. Should be fixed now!